src/Dialect/ONNX/ONNXOps.td.inc

```diff
@@ -4803,6 +4803,7 @@ def ONNXRoiAlignOp:ONNX_Op<"RoiAlign",
       return {20};
     }
   }];
```

Comment:
We typically don't modify these machine generated files by hand. 

The file that drive the generation of the .inc file is this one: utils/gen_onnx_mlir.py . Search for `OpsWithVerifier` in that file and add the two ops there. Explanation here: https://github.com/onnx/onnx-mlir/blob/main/docs/HowToAddAnOperation.md#generate-the-proper-onnxtdinc

The inc files can be regenerated using the command is `make OMONNXOpsIncTranslation `. If you work in a Docker env, it will work, otherwise look for installation step needed for ONNX here. https://github.com/onnx/onnx-mlir/blob/main/docs/ImportONNXDefs.md#how-to-use-the-script

In your case, since you manually edited the inc file, you can probably skip that step

Reply:
Thanks for the wise suggestion, revised.